pkg/istio/authorizationpolicy/controller.go

```diff
@@ -369,3 +387,41 @@ func (c *Controller) checkAuthzEnabledAnnotation(serviceObj *corev1.Service) boo
 	}
 	return false
 }
+
+// cleanUpStaleAP deletes the existing Authorization Policy associated to the service which is switching back from
+// Authorization Policy Enabled back to SR/SRB
+func (c *Controller) cleanUpStaleAP() error {
+	// Fetch the Authorization Policies present across all the namespaces
+	currentAPList, err := c.configStoreCache.List(collections.IstioSecurityV1Beta1Authorizationpolicies.Resource().GroupVersionKind(), "")
+	if err != nil {
+		return fmt.Errorf("Error while fetching the Authorization Policy resources from cache : %v", err.Error())
+	}
+
+	for _, currAP := range currentAPList {
+		nsSvcArr := strings.Split(currAP.Key(), "/")
+		serviceName := nsSvcArr[2]
+		serviceNamespace := nsSvcArr[1]
+		key := serviceNamespace + "/" + serviceName
```

Comment:
Can we just use `currApp.Namespace + "/" + currApp.Name` instead so we don't have to do any string manipulation?